src/vs/editor/contrib/hover/browser/contentHover.ts

```diff
@@ -68,16 +67,22 @@ export class ContentHoverController extends Disposable {
 		}));
 		this._register(TokenizationRegistry.onDidChange(() => {
 			if (this._widget.position && this._currentResult) {
-				this._widget.clear();
 				this._setCurrentResult(this._currentResult); // render again
 			}
 		}));
 	}
 
+	get widget() {
+		return this._widget;
+	}
+
 	/**
 	 * Returns true if the hover shows now or will show.
 	 */
 	public maybeShowAt(mouseEvent: IEditorMouseEvent): boolean {
+		if (this._widget.isResizing) {
+			return true;
```

Comment:
Similar to hiding and resizing. Is it that during resize we get show/hide requests?

Reply:
This method is called inside of _onMouseMove to decide whether the mouse is showing or will show. In the case when the widget is resizing, the hover is already showing.